honeycomb-guide/src/index.md

```diff
@@ -13,13 +13,38 @@ issues, thanks to the language's guarantees.
 
 ### Requirements
 
-- **Rust 1.75** - *The code may compile and work for earlier versions, but we do not test for those*
+- **Rust 1.76** - *The code may compile and work for earlier versions, but we do not test for those*
```

Comment:
Can we say, the current stable version? Or do you think you will update this explicit version on a regular basis?

Reply:
Good point, I'll change it to stable